operator/v1/types_ingress.go

```diff
@@ -716,8 +720,58 @@ type AWSNetworkLoadBalancerParameters struct {
 	// +optional
 	// +openshift:enable:FeatureGate=IngressControllerLBSubnetsAWS
 	Subnets *AWSSubnets `json:"subnets,omitempty"`
+
+	// eipAllocations assign Elastic IP addresses to the Network Load Balancer.
+	// eipAllocations cannot be used with "Internal" scope.
+	// An EIP can be allocated to only a single IngressController.
+	// The number of Allocation IDs must match the number of subnets that are used for the load balancer.
+	// See https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/elastic-ip-addresses-eip.html for general
+	// information about configuration, characteristics, and limitations of Elastic IP addresses.
+	// +openshift:enable:FeatureGate=SetEIPForNLBIngressController
+	// +optional
+	// +listType=atomic
+	// Each entry in the eipAllocations list must be unique
+	// +kubebuilder:validation:XValidation:rule=`self.all(x, self.exists_one(y, x == y))`,message="eipAllocations cannot contain duplicates"
+	// A maximum of 10 IP allocations are permitted
+	// +kubebuilder:validation:MaxItems=10
+	EIPAllocations []EIPAllocation `json:"eipAllocations"`
 }
 
+// EIPAllocation is a reference to an EIP that can be allocated to an ELB in the AWS environment.
+// The pattern for an EIP allocation ID is as follows:
+//
+// Starts with eipalloc-
+//
+// # Followed by an alphanumeric string of exactly 17 characters
```

Comment:
Is the "#" intentional?  What does it signify?  It looks like it is rendered as a literal "#" in the generated yaml.

Reply:
i removed it.